client/src/components/bundle/Collapsed.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "styled-components";
+
+import { urlFor } from "../../util";
+import BundleContentProps from "./BundleContentProps";
+
+const CollapsedRoot = styled.div`
+  height: 300px;
+  width: 350px;
+  background-color: #282c34;
+  margin: 1%;
+  display: block;
+  box-shadow: 0 4px 8px 0 rgba(255, 255, 255, 0.2),
+    0 6px 20px 0 rgba(247, 244, 244, 0.19);
+`;
+
+const CollapsedFlex = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: inherit;
+  height: 29%;
+`;
+
+const Title = styled.div.attrs({
+  children: "Pink Butterfly",
+})`
+  width: 50%;
+  padding: 0.5em;
+  text-align: center;
+  font-size: large;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  color: white;
+`;
+
+const ImgService = styled.img.attrs({
+  src: urlFor("image_placeholder.jpg"),
+  alt: "Some image",
+})`
+  height: 70%;
+  width: inherit;
+  background-color: #282c34;
+  margin: none;
+`;
+
+const BigAssButton = styled.button`
+  width: 30%;
+  height: 50%;
+  margin-left: 2.1em;
+  align-self: center;
+  border-radius: 4px;
+  background-color: inherit;
+  color:white;
+`;
+
+const Collapsed = ({ open }: BundleContentProps): JSX.Element => (
```

Comment:
Here you have annotated the props with the `BundleContentProps` type, but you are not using 2/3rds of the props in that type. It would be better to use an inline annotation here, or to make another type. Like so:


```ts
const Collapsed = (
  { open }: { open?: MouseEventHandler<HTMLButtonElement> } = {
    open: () => undefined,
  }
): JSX.Element => (
```

Or like this:

```ts
type CollapsedProps = { open?: MouseEventHandler<HTMLButtonElement> };
const Collapsed = (
  { open }: CollapsedProps = { open: () => undefined }
): JSX.Element => (
```

Reply:
This way it would show the actual props that are used by this component

Reply:
Resolved